getting_started/cloud_infrastructure_access/dedicated-aws-access.adoc

```diff
@@ -0,0 +1,56 @@
+[id="dedicated-aws-access"]
+= AWS Infrastructure Access
+include::modules/common-attributes.adoc[]
+:context: dedicated-aws-access
+
+AWS infrastructure access allows link:https://access.redhat.com/node/3610411[Customer Portal Organization Admins] and cluster owners to enable Amazon Web Services (AWS) Identity and Access Management (IAM) users to have federated access to the AWS Console for their {product-title} cluster. Admins may select between Network Management or Read-only access options.
+
+[id="dedicated-configuring-aws-access"]
+== Configuring AWS Infrastructure Access
+
+.Prerequisites
+. An existing AWS account. You may create a new AWS account by following link:https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/[AWS documentation].
+
+. An IAM user with `STS:AllowAssumeRole` permissions within the AWS account.
+.. Open the IAM dashboard of the AWS Console.
+.. In the *Policies* section, click *Create Policy*.
+.. Select the `JSON` tab and replace the existing text with the following:
++
+[source,json]
+----
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "*"
+        }
+    ]
+  }
+----
+.. Click *Review Policy*.
+.. Provide an appropriate name and description, then click *Create Policy*.
+.. In the *Users* section, click *Add user*.
+.. Provide an appropriate user name.
+.. Select *AWS Management Console access* and other roles as needed.
+.. Adjust the password requirements as necessary for your organization, then click *Next: Policy*.
+.. Click the *Attach existing policies directly* option.
+.. Search for and check the policy created in previous steps.
++
+[NOTE]
+====
+It is not recommended to set a permissions boundary.
+====
+.. Click *Next: Tags*, then click *Next: Review*. Confirm the configuration is correct.
+.. Click *Create user*, then click *Close* on the success page.
+. The IAM user's Amazon Resource Name (ARN) will have the following format: `arn:aws:iam::000111222333:user/username`.
+
+.Procedure
+. Open the {product-title} Cluster Manager in your browser and select the cluster you wish to allow AWS infrastructure access.
+. Select the *Access control* tab, and scroll to the *AWS Infrastructure Access* section.
+. Paste the AWS IAM ARN and select `Network Management` or `Read-only` permissions, then click *Grant role*.
+. Copy the AWS OSD Console URL to your clipboard.
+. Sign in to your AWS account with your Account ID or alias, IAM user name, and password.
+. In a new browser tab, paste the AWS OSD Console URL to be routed to the AWS Switch Role page.
```

Comment:
This makes it sound like you will be routed when you paste the URL, which I don't think is going to happen. :) Suggest something like, "paste the AWS OSD Console URL that will be used to route to the AWS Switch Role page."